e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/ToolSettings.java

```diff
@@ -122,6 +123,24 @@ public ToolSettings setDefaultCommand(final String command) {
         return this;
     }
 
+    public ToolSettings disableAllowSensitiveStorage() {
+        SelenideElement checkbox = context().$(byText("Allow sensitive storages"))
+                .parent().find(By.xpath("following-sibling::div//span"));
```

Comment:
It would be better to add the checkbox to Map<Primitive, SelenideElement> elements

Reply:
Done